keyboards/lily58/keymaps/paulomp90/config.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2023 Paulo Pereira
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define MASTER_LEFT
```

Comment:
Master left isn't needed here, either.  The default behavior is master left, and defaults to that if no other handedness defines are present. 

```suggestion
```

Though, personally, I prefer ee hands, if split pin isn't supported, as either side can be used, that way.

Reply:
Thanks @drashna